docs/user/getting-started.md

```diff
@@ -259,7 +346,7 @@ RUN dnf -y copr enable @redhat-et/flightctl-dev centos-stream-9-x86_64 && \
     dnf -y clean all; \
     systemctl enable flightctl-agent.service
 
-ADD config.yaml /etc/flightctl/
+ADD agentconfig.yaml /etc/flightctl/
```

Comment:
I believe the agent is still looking for `config.yaml`, I did something similar myself and the agent failed to find the file.

```suggestion
ADD agentconfig.yaml /etc/flightctl/config.yaml
```

Reply:
Thank you @celdrake !